packages/editor-ui/src/mixins/workflowHelpers.ts

```diff
@@ -69,6 +69,357 @@ import { ICredentialsResponse } from '@/Interface';
 let cachedWorkflowKey: string | null = '';
 let cachedWorkflow: Workflow | null = null;
 
+export function resolveParameter(
```

Comment:
Changes to this file are only moving resolution-related functions out of the mixin so they can be called during completion.